lib/signal_tower/prometheus_stats.ex

```diff
@@ -4,20 +4,41 @@ defmodule SignalTower.PrometheusStats do
   @counter [name: :palava_room_closed_total, labels: [], help: "Number of rooms closed"]
   @counter [name: :palava_joined_room_total, labels: [], help: "Number of peers joined a room"]
   @counter [name: :palava_leave_room_total, labels: [], help: "Number of peers left a room"]
+  @histogram [
+    name: :palava_duration_room_milliseconds,
+    labels: [],
+    buckets: Prometheus.Buckets.new({:exponential, 1000, 2, 20}),
```

Comment:
just out of curiosity, what does the {:exponential, 1000, 2, 20} stand for? maybe a comment would be good?

Reply:
The buckets starts with 1000 and are multiplied by 2 for 20 buckets e.g.: 1000, 2000, 4000, 8000... I will add a comment,